src/driver/RealtimeDatabase/InProcessRealtimeDatabase.ts

```diff
@@ -89,6 +92,7 @@ class InProcessRealtimeDatabaseRef implements IFirebaseRealtimeDatabaseRef {
     constructor(
         private readonly db: InProcessRealtimeDatabase,
         private readonly path: string,
+        private readonly idGenerator: IdGenerator = nanoid,
```

Comment:
I was hoping to match the natural ordering behaviour that Realtime Database has on `push()`, so maybe default to something other than nanoid? Or we could prefix nanoid with the unix timestamp including ms?